ui/desktop/src/bin/npx

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Enable strict mode to exit on errors and unset variables
+set -euo pipefail
+
+# Set log file
+LOG_FILE="/tmp/mcp.log"
+
+# Clear the log file at the start
+> "$LOG_FILE"
+
+# Function for logging
+log() {
+    local MESSAGE="$1"
+    echo "$(date +'%Y-%m-%d %H:%M:%S') - $MESSAGE" | tee -a "$LOG_FILE"
+}
+
+# Trap errors and log them before exiting
+trap 'log "An error occurred. Exiting with status $?."' ERR
+
+log "Starting npx setup script."
+
+# Ensure ~/.goose/mcp-hermit/bin exists
+log "Creating directory ~/.goose/mcp-hermit/bin if it does not exist."
+mkdir -p ~/.goose/mcp-hermit/bin
```

Comment:
We've historically used `~/.config/goose` I think. Thoughts on that or this?

Reply:
yes - could shift to that